samples/fabric/simpleassettransfer/tokenasset_test.go

```diff
@@ -569,8 +569,8 @@ func TestReclaimTokenAsset(t *testing.T) {
 	tokenClaimStatus.ExpirationStatus = false
 	tokenClaimStatusBytes, _ = marshalAssetClaimStatus(tokenClaimStatus)
 	err = simpleAsset.ReclaimTokenAsset(transactionContext, pledgeId, getRecipientECertBase64(), destNetworkID, tokenClaimStatusBytes)
-	require.EqualError(t, err, fmt.Sprintf("cannot reclaim asset with pledgeId %s as the pledge has not yet expired", pledgeId))       // claim probe time was before expiration time
-	
+	require.EqualError(t, err, fmt.Sprintf("cannot reclaim asset with pledgeId %s as the claim has not yet expired", pledgeId))       // claim probe time was before expiration time
```

Comment:
Why does the error message say `claim has not expired`, correct sentence would be `pledge has not expired` right?

Reply:
Ahh..you are right. I misunderstood this. Will revert the code and the unit tests.